### PR TITLE
СС-18569 Fix docs - Switch from Yarn to NPM

### PR DESCRIPTION
**Pull request by danilmoro3**
Ticket: https://spryker.atlassian.net/browse/CC-18569

### CODE REVIEW

File: docs/scos/dev/front-end-development/migration-guide-switch-from-yarn-to-npm.md [hunk 1]
```diff
@@ -41,7 +41,22 @@ To ensure the CI jobs will run successfully, add the same config part to all 'de
...
 
-2. Run the following command to pull the latest `docker-sdk` version:
+2. The following commands are deprecated and will work correctly only with the previous versions (using Yarn). Please check and remove deprecated commands in all `*.yml` files.
+
+```bash
+frontend:yves:install-dependencies
+frontend:zed:install-dependencies
+```
+
+And replace `frontend:mp:install-dependencies` with the next one `frontend:project:install-dependencies`.
```
Comment (FRITS-Kh) [inline docs/scos/dev/front-end-development/migration-guide-switch-from-yarn-to-npm.md:51]:
```suggestion
2. The following commands are deprecated and will work correctly only with the previous versions (using Yarn). 
Please check all `*.yml` files at the project level and:
- Remove deprecated `frontend:yves:install-dependencies` and `frontend:zed:install-dependencies` commands.
- Replace the `frontend:mp:install-dependencies` command with the next one `frontend:project:install-dependencies`.
```

**Review by FRITS-Kh — CHANGES_REQUESTED** [review]

**Review by FRITS-Kh — DISMISSED** [review]

File: docs/scos/dev/front-end-development/migration-guide-switch-from-yarn-to-npm.md [hunk 2]
```diff
@@ -41,7 +41,18 @@ To ensure the CI jobs will run successfully, add the same config part to all 'de
 
 {% endinfo_block %}
 
-2. Run the following command to pull the latest `docker-sdk` version:
+2. The following commands are deprecated and will work correctly only with the previous versions (using Yarn).
```
Comment (VadymSachenko) [inline docs/scos/dev/front-end-development/migration-guide-switch-from-yarn-to-npm.md:44]: What do you mean by "the following commands"? Only commands from lines 46 and 47 below?
Remove deprecated `frontend:yves:install-dependencies` and `frontend:zed:install-dependencies` commands.
Replace the `frontend:mp:install-dependencies` command with the next one `frontend:project:install-dependencies`.

Comment (danilmoro3) [reply docs/scos/dev/front-end-development/migration-guide-switch-from-yarn-to-npm.md:44]: These commands are deprecated 
```
frontend:yves:install-dependencies
frontend:zed:install-dependencies
frontend:mp:install-dependencies
```

Now we are using only one command to install all dependencies
```
frontend:project:install-dependencies
```

File: docs/scos/dev/front-end-development/migration-guide-switch-from-yarn-to-npm.md [hunk 3]
```diff
@@ -37,18 +37,28 @@ image:
...
+
+4. Pull the latest `docker-sdk` version:
 
 ```bash
 cd docker && git pull origin master
 git log
 ```
 
-`git log` command will print the table with the last three commits, copy the `hash` of the first commit and replace content in the `.git.docker` file:
+The `git log` command prints the table with the last three commits, copies the `hash` of the first commit, and replaces the content in the `.git.docker` file:
```
Comment (danilmoro3) [inline docs/scos/dev/front-end-development/migration-guide-switch-from-yarn-to-npm.md:61]: This command only prints commits, need to copy paste hash manually

**Review by FRITS-Kh — APPROVED** [review]

**Review by VadymSachenko — APPROVED** [review]

**Pull request merged**